src/components/Card/Card.css

```diff
@@ -1,11 +1,25 @@
+.button-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.button-refresh {
+  border: none;
+  background-color: inherit;
+  font-size: 22px;
+  padding: 0;
+  margin: 0;
+}
+
```

Comment:
Looks great for getting our basic functionality incorporated, and we'll be getting into more developed styling in our next round of work.